source/tutorial/connecting-to-atlas.txt

```diff
@@ -18,6 +18,13 @@ Atlas uses TLS/SSL to encrypt connections and enforces authentication by
 default. This page explains how to connect to Atlas using
 :program:`mongodrdl` and :program:`mongosqld`.
 
+.. note::
+   
+   With the MongoDB Atlas free tier, |bi| cannot use the ``allowDiskUse``
```

Comment:
I'd like to make allowDiskUse a link, but it would need to be coordinated with a manual tweak, so let's not block on that at the moment